### DISCUSSION TITLE
Add a new i18n key easely in all language files

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3930] [Q&A] [answered]
Hi,

when I need a new i18n key, than I have to add it to all 17 languages.
Is there a easy way so set the English i18n string as default to all other languages?

**Comment by Alkarex** [comment]
```sh
cli/manipulate.translation.php -a format
```

Also available via composer or make.

**Comment by aledeg** [comment, marked as answer]
```
cli/manipulate.translation.php -a add -k new.key -v new.value
```

check the script help to see every possible actions.
